src/Components/Authorization/AuthForm.tsx

```diff
@@ -0,0 +1,109 @@
+import React from "react"
+
+import ForgotPasswordForm from "./ForgotPasswordForm"
+import LoginForm from "./LoginForm"
+import RegisterForm from "./RegisterForm"
+import { FormComponentType, FormFields, Mode } from "./Types"
+
+type InputValidator = (val: string) => string
+
+interface Props extends React.HTMLProps<HTMLDivElement> {
```

Comment:
Is this going to be the form itself? Wondering if the props here should extends `<HTMLFormElement>` vs `<HTMLDivElement>`?

Reply:
Just noticed you are pulling in some of the form types. Maybe you can extend the props there.

Reply:
Ah yes, good point.